lib/cc/analyzer/formatters/plain_text_formatter.rb

```diff
@@ -56,7 +56,11 @@ def finished
 
         def engine_running(engine)
           @active_engine = engine
-          with_spinner("Running #{engine.name}: ") do
+          if $stdout.tty?
+            with_spinner("Running #{engine.name}: ") do
```

Comment:
What do you think of putting this logic inside of `Spinner`? The `start` method could become:

        def start
          if $stdout.tty?
            @thread = Thread.new do
              loop do
                @spinning = true
                spinner.spin
                sleep 0.075
              end
            end
          end
        end